pkg/nsx/services/firewall.go

```diff
@@ -116,7 +151,7 @@ func (service *SecurityPolicyService) CreateOrUpdateSecurityPolicy(obj *v1alpha1
 		return err
 	}
 
-	// The method Operate*CR* knows how to deal with CR, if there is MarkedForDelete, then delete it from store,
+	// The steps below knows how to deal with CR, if there is MarkedForDelete, then delete it from store,
```

Comment:
The steps below knows -> The steps below know

Reply:
ok